DragaliaAPI/Features/Emblem/EmblemController.cs

```diff
@@ -0,0 +1,48 @@
+﻿using DragaliaAPI.Controllers;
+using DragaliaAPI.Database.Repositories;
+using DragaliaAPI.Models;
+using DragaliaAPI.Models.Generated;
+using DragaliaAPI.Services;
+using DragaliaAPI.Services.Exceptions;
+using Microsoft.AspNetCore.Mvc;
+
+namespace DragaliaAPI.Features.Emblem;
+
+[Route("emblem")]
+public class EmblemController(
+    IEmblemRepository emblemRepository,
+    IUserDataRepository userDataRepository,
+    IUpdateDataService updateDataService
+) : DragaliaControllerBase
+{
+    [HttpPost("get_list")]
+    public async Task<DragaliaResult> GetList()
+    {
+        EmblemGetListData resp = new();
+
+        resp.emblem_list = (await emblemRepository.GetEmblemsAsync()).Select(
```

Comment:
Do we need the mapper profile if we aren't using mapper to go to EmblemList

Reply:
I added the mapper profile before realizing the emblem list is in neither load/index nor the update data list, will remove it